app/src/routes/project/Canvas/index.tsx

```diff
@@ -1,18 +1,21 @@
-import { ReactNode, useEffect, useRef, useState } from 'react';
+import { EditorMode } from '@/lib/models';
+import { isMetaKey } from '@/lib/utils';
+import { ReactNode, useCallback, useEffect, useRef, useState } from 'react';
 import { useEditorEngine } from '..';
 import PanOverlay from './PanOverlay';
 
 const Canvas = ({ children }: { children: ReactNode }) => {
     const [position, setPosition] = useState({ x: 300, y: 50 });
     const [scale, setScale] = useState(0.6);
+    const [isPanning, setIsPanning] = useState(false);
 
     const editorEngine = useEditorEngine();
     const containerRef = useRef<HTMLDivElement>(null);
     const zoomSensitivity = 0.006;
     const panSensitivity = 0.52;
 
     const handleWheel = (event: WheelEvent) => {
-        if (event.ctrlKey) {
+        if (event.ctrlKey || event.metaKey) {
```

Comment:
When pinching on trackpad, ctrlKey is enabled on Mac. I think this is silly as a standard but we need this for zoom on track pad to work.